nkms/network/server.py

```diff
@@ -62,14 +62,14 @@ def serialize_capabilities(self):
 
     def get_now(self, key):
```

Comment:
Do you think we could rename this to something more self documenting?

Reply:
Hmm - sure!  I just took `get` and made it happen synchronous / blocking (ie, "get it *now*").  

What do you suggest instead?

Reply:
Ah, I guess that makes sense then. It just feels weird to simply see `self.get` or `self.get_now`.

I think I prefer `get_<thing>_now` so it's more clear, but this is fine too.

Reply:
Yeah... I'm just not sure what to make <thing>.  :-)

`get_value_now`?  I'm not sure if `value` adds anything to me or not, but I guess it does reduce the initial cryptic phrasing.